com.github.Bleuzen.FFaudioConverter.appdata.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.github.Bleuzen.FFaudioConverter</id>
+  <launchable type="desktop-id">ffaudioconverter.desktop</launchable>
+  <project_license>MIT</project_license>
+  <name>FFaudioConverter</name>
+  <summary>Convert audio files</summary>
+  <description>
+    <p>Qt GUI for converting audio files with FFmpeg</p>
```

Comment:
Nit pick that this is probably not a great description for users. Mentioning the names of libraries is frowned upon as just technical jargon.

Reply:
Ok I have very simplified it.

Reply:
Looks like it was just removed?

Reply:
yeah, only the categories.

Reply:
Oh ok, I added the description back.